release-notes.txt

```diff
@@ -0,0 +1,235 @@
+BISDN Linux
+v4.5.0 Release Notes
+Date: 2022-03-15
+
+! New in this release
+- baseboxd:
+  - updated version to 1.11.1
+  - added initial support for Broadcom KNET interfaces.
+- Linux kernel updated to version 5.15.22
+- iproute2: udpate to version 5.16
```

Comment:
i think we also added support for STP on bonded interfaces (https://github.com/bisdn/bisdn.github.io/pull/136)

Reply:
and we fixed this bug here: https://github.com/bisdn/basebox/issues/364

Reply:
Mentioned STP support on bonded interfaces, and added the bug fix to list of bugfixes